Formula/rio.rb

```diff
@@ -0,0 +1,28 @@
+class Rio < Formula
+  desc "A hardware-accelerated GPU terminal emulator powered by WebGPU, focusing to run in desktops and browsers."
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  version "0.0.7"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "pkg-config" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "rustup-init" => :test
+
+  on_linux do
+    depends_on "mesa"
+  end
+
+  def install
+    system "make build"
+    bin.install "target/release/rio"
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```

Reply:
would that lead to use `--release` flag as well?

Reply:
Rio is a cargo workspace repo, so I suppose will be

```rb
cd "rio" do
  system "cargo", "install", *std_cargo_args
end
```

Reply:
> would that lead to use `--release` flag as well?

It will, yes.



> Rio is a cargo workspace repo, so I suppose will be
> 
> ```ruby
> cd "rio" do
>   system "cargo", "install", *std_cargo_args
> end
> ```

Try
```ruby
system "cargo", "install", *std_cargo_args(path: "rio")
```

Reply:
updated 3b2ab9c2fb0622cfac77332e21f70c6b8c715b2d